CONTRIBUTING.md

```diff
@@ -21,6 +21,9 @@ That said, please do file any bugs you find, keeping the following in mind:
 * One issue per bug. Putting multiple things in the same issue makes both discussion and completion unnecessarily complicated.
 * No build issues (or other support requests). If the GitHub Actions CI build succeeds, the build problem is most likely on your side. Work it out locally, or ask in the `#build-problems` channel on Discord.
 * Don't comment on issues just to add a joke or irrelevant commentary. Hundreds of people get notified about comments so let's keep them relevant.
+* For bare metal issues - please include the debug log from serial console, what you tried to do to solve the issue before 
```

Comment:
```suggestion
* For bare metal issues, please include the complete debug log from the serial console, what you tried to do to solve the issue before 
```

to avoid "surely this part of the log is not important..."

Reply:
Yeah, let's not deal with people overthinking if they should include some parts of the log or not :sweat_smile: